sklearn/datasets/covtype.py

```diff
@@ -118,16 +118,16 @@ def fetch_covtype(data_home=None, download_if_missing=True,
         X = Xy[:, :-1]
         y = Xy[:, -1].astype(np.int32)
 
-        joblib.dump(X, samples_path, compress=9)
-        joblib.dump(y, targets_path, compress=9)
+        dump(X, samples_path, compress=9)
+        dump(y, targets_path, compress=9)
```

Comment:
Sorry to do this, but again I think `joblib.dump` was clearer where,
```py
from ..utils import _joblib as joblib
```
here and below.

Because there is `pickle.dump` it's hard to know what this function calls when reading the code.